argilla-server/src/argilla_server/api/schemas/v1/fields.py

```diff
@@ -77,14 +75,17 @@ class ImageFieldSettingsUpdate(BaseModel):
 
 class ChatFieldSettings(BaseModel):
     type: Literal[FieldType.chat]
+    use_markdown: bool = True  # Just for dev purposes. Otherwise getting old datasets will fail
```

Comment:
I guess the following change should be done before merging:
```suggestion
    use_markdown: bool
```

Reply:
Yep. It was a workaround to support reading already created datasets in the dev environment